redbot/cogs/audio/core/tasks/startup.py

```diff
@@ -33,17 +43,109 @@ async def initialize(self) -> None:
+                            break
+                        except IndexError:
+                            await asyncio.sleep(5)
+                            tries += 1
+                        except Exception as exc:
+                            debug_exc_log(log, exc, "Failed to restore music voice channel")
+                            if vc is None:
+                                break
+
+                if tries >= 25 or guild is None or vc is None:
+                    await self.api_interface.persistent_queue_api.drop(guild_id)
+                    continue
+
+                shuffle = await self.config.guild(guild).shuffle()
+                repeat = await self.config.guild(guild).repeat()
+                volume = await self.config.guild(guild).volume()
+                shuffle_bumped = await self.config.guild(guild).shuffle_bumped()
+                player.repeat = repeat
+                player.shuffle = shuffle
+                player.shuffle_bumped = shuffle_bumped
+                if player.volume != volume:
+                    await player.set_volume(volume)
+                for track in track_data:
+                    track = track.track_object
+                    player.add(guild.get_member(track.extras.get("requester")) or guild.me, track)
+                player.maybe_shuffle()
+
+                await player.play()
+            except Exception as err:
+                debug_exc_log(log, err, f"Error restoring player in {guild_id}")
+                await self.api_interface.persistent_queue_api.drop(guild_id)
+
+    async def maybe_message_all_owners(self):
+        current_notification = await self.config.owner_notification()
+        if current_notification == _OWNER_NOTIFICATION:
+            return
+        if current_notification < 1 <= _OWNER_NOTIFICATION:
+            msg = _(
+                """Hey, first of all sorry for this notification, it is not something we wish to do often but this update to the Audio Cog warrants it in this case.
+            
+Audio version 2.3.0+ brings you access to the **Global Audio API**, `[p]help audioset globalapi` for more info, this **API is disabled by default and will never be enabled until you manually enable it**.
+
+This API will allow your to use the Spotify functionally of Audio with a much smaller YouTube API key usage, as it will attempt to look for results in the API before using your key.
+This API will also help your bot by reducing the likelihood of YouTube rate-limiting your bot for making requests too often.
+
+To use this API **you will be required to have an access token**, to obtain this token please join <https://discord.gg/red> and run `?audioapi register` in #testing.
+Note: Just like any request your bot makes this service will be able to see your bot IP, your bot IP is not used for anything in the server but we felt that disclosing this to potential users is an important step since the service is managed by Draper and Cog-Creators org.
```

Comment:
```suggestion
Note: Just like any request your bot makes, this service will be able to see your bot IP. Your bot IP is not used for anything in the server but we felt that disclosing this to potential users is an important step since the service is managed by Draper and Cog-Creators org.
```